app/test_split_integer.py

```diff
@@ -2,20 +2,33 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    initial_value = 32
+    number_of_parts = 6
+    assert (sum(split_integer(initial_value, number_of_parts))
+            == initial_value)
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    initial_value = 16
```

Comment:
if you reuse same parameters  in different test they can be placed into a fixture